README.md

```diff
@@ -35,40 +39,41 @@ From the wolfSSH source directory run:
     $ make
     $ make check
 
-The `autogen.sh` script only has to be run the first time after cloning the
-repository. If you have already run it or are using code from a source
-archive, you should skip it.
+The `autogen.sh` script only has to be run the first time after cloning
+the repository. If you have already run it or are using code from a
+source archive, you should skip it.
 
 For building under Windows with Visual Studio, see the file
 "ide/winvs/README.md".
 
-NOTE: On resource constrained devices the DEFAULT_WINDOW_SZ may need to be set
-to a lower size. It can also be increased in desktop use cases to help with
-large file transfers. By default channels are set to handle 16,384 bytes of data
-being sent and received. An example of setting a window size for new channels
-would be as follows "./configure CPPFLAGS=-DDEFAULT_WINDOW_SZ=16384"
+NOTE: On resource constrained devices the `DEFAULT_WINDOW_SZ` may need
+to be set to a lower size. It can also be increased in desktop use cases
+to help with large file transfers. By default channels are set to handle
+16,384 bytes of data being sent and received. An example of setting a
+window size for new channels would be as follows
+`./configure CPPFLAGS=-DDEFAULT_WINDOW_SZ=16384`
 
 examples
 --------
 
-The directory `examples` contains an echoserver that any client should be able
-to connect to. From the terminal run:
+The directory `examples` contains an echoserver that any client should
+be able to connect to. From the terminal run:
 
     $ ./examples/echoserver/echoserver -f
 
 The option `-f` enables echo-only mode. From another terminal run:
 
-    $ ssh_client jill@localhost -p 22222
+    $ ssh jill@localhost -p 22222
```

Comment:
optional: provide a link to the ssh homepage so reader knows where to download.

Reply:
It is typically already on everybody's computer. Except for Windows people, I guess.

Reply:
True enough.